graphql/schemabuilder/pagination.go

```diff
@@ -676,6 +706,14 @@ func getFuncReturnType(fn interface{}) reflect.Type {
 	return typ.Out(0)
 }
 
+func getFuncReturnTypeBatched(fn interface{}) reflect.Type {
```

Comment:
is this called?  I can't find any other references to this function in the commit